.github/workflows/ci.yaml

```diff
@@ -54,12 +54,12 @@ jobs:
         # TODO: Replace with custom image on self-hosted runner
         uses: charmed-kubernetes/actions-operator@main
         with:
-          juju-channel: 3.1/stable
+          juju-channel: 3.4/stable
           provider: microk8s
           channel: 1.28-strict/stable
           microk8s-group: snap_microk8s
           microk8s-addons: "hostpath-storage dns"
-          bootstrap-options: "--agent-version 3.1.6"
+          # bootstrap-options: "--agent-version 3.4.3"
```

Comment:
**question:** Why comment? Is it not needed? I guess not for K8s.

Reply:
I was just testing if pinning was required (not sure why we were pinning for the 3.1)